edgraph/server_test.go

```diff
@@ -204,4 +204,7 @@ func TestParseSchemaFromAlterOperation(t *testing.T) {
 		})
 	}
 
+	ps.Close()
+	os.RemoveAll(dir)
```

Comment:
I am a little confused here, does this cleanup happen even when the tests fail? I am trying to understand if this is part of the @after state or @test state.

Reply:
https://pkg.go.dev/testing#hdr-Subtests_and_Sub_benchmarks

Reply:
@skrdgraph Thanks for noticing this, I think now it is much cleaner.  Note that defer statements are executed in[ FILO order.](https://go.dev/blog/defer-panic-and-recover)